### ISSUE TITLE
Fails to setup on port 53

### ISSUE THREAD

**Issue by r-saikat** [angristan/wireguard-install#190]
The wireguard server fails to be properly set up on a new VM (tried on oracle cloud, GCP, Digital Ocean). UDP 53 should be a good option to evade port blocking at public wifi/ universities. 

The following error appears while configuring the server:

```
* Applying /etc/sysctl.conf ...
Job for wg-quick@wg0.service failed because the control process exited with error code.
See "systemctl status wg-quick@wg0.service" and "journalctl -xe" for details.
Created symlink /etc/systemd/system/multi-user.target.wants/wg-quick@wg0.service → /lib/systemd/system/wg-quick@.service.

WARNING: WireGuard does not seem to be running.
You can check if WireGuard is running with: systemctl status wg-quick@wg0
If you get something like "Cannot find device wg0", please reboot!
```
The following appears just after creating a peer:

```
Job for wg-quick@wg0.service failed because the control process exited with error code.
See "systemctl status wg-quick@wg0.service" and "journalctl -xe" for details.
```

**Comment by r-saikat** [comment]
I made sure I am root and nothing else is running on port 53. This is a brand new VM. (Ubuntu 20.04.2 LTS) The following appears when I run `systemctl status wg-quick@wg0.service`

```
● wg-quick@wg0.service - WireGuard via wg-quick(8) for wg0
     Loaded: loaded (/lib/systemd/system/wg-quick@.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sun 2021-02-14 18:21:18 UTC; 11min ago
       Docs: man:wg-quick(8)
             man:wg(8)
             https://www.wireguard.com/
             https://www.wireguard.com/quickstart/
             https://git.zx2c4.com/wireguard-tools/about/src/man/wg-quick.8
             https://git.zx2c4.com/wireguard-tools/about/src/man/wg.8
    Process: 28488 ExecStart=/usr/bin/wg-quick up wg0 (code=exited, status=2)
   Main PID: 28488 (code=exited, status=2)

Feb 14 18:21:17 test wg-quick[28488]: [#] ip link add wg0 type wireguard
Feb 14 18:21:17 test wg-quick[28488]: [#] wg setconf wg0 /dev/fd/63
Feb 14 18:21:17 test wg-quick[28488]: [#] ip -4 address add 10.66.66.1/24 dev wg0
Feb 14 18:21:17 test wg-quick[28488]: [#] ip -6 address add fd42:42:42::1/64 dev wg0
Feb 14 18:21:17 test wg-quick[28488]: [#] ip link set mtu 8920 up dev wg0
Feb 14 18:21:17 test wg-quick[28530]: RTNETLINK answers: Address already in use
Feb 14 18:21:17 test wg-quick[28488]: [#] ip link delete dev wg0
Feb 14 18:21:18 test systemd[1]: wg-quick@wg0.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Feb 14 18:21:18 test systemd[1]: wg-quick@wg0.service: Failed with result 'exit-code'.
Feb 14 18:21:18 test systemd[1]: Failed to start WireGuard via wg-quick(8) for wg0.

```

And this appears when I run `journalctl -xe`

```
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- An ExecStart= process belonging to unit wg-quick@wg0.service has exited.
--
-- The process' exit code is 'exited' and its exit status is 2.
Feb 14 18:21:18 test systemd[1]: wg-quick@wg0.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- The unit wg-quick@wg0.service has entered the 'failed' state with result 'exit-code'.
Feb 14 18:21:18 test systemd[1]: Failed to start WireGuard via wg-quick(8) for wg0.
-- Subject: A start job for unit wg-quick@wg0.service has failed
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A start job for unit wg-quick@wg0.service has finished with a failure.
--
-- The job identifier is 2246 and the job result is failed.
Feb 14 18:26:04 test PackageKit[1996]: daemon quit
Feb 14 18:26:04 test systemd[1]: packagekit.service: Succeeded.
-- Subject: Unit succeeded
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- The unit packagekit.service has successfully entered the 'dead' state.
Feb 14 18:28:41 test systemd[1]: Starting Cleanup of Temporary Directories...
-- Subject: A start job for unit systemd-tmpfiles-clean.service has begun execution
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A start job for unit systemd-tmpfiles-clean.service has begun execution.
--
-- The job identifier is 2409.
Feb 14 18:28:41 test systemd[1]: systemd-tmpfiles-clean.service: Succeeded.
-- Subject: Unit succeeded
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- The unit systemd-tmpfiles-clean.service has successfully entered the 'dead' state.
Feb 14 18:28:41 test systemd[1]: Finished Cleanup of Temporary Directories.
-- Subject: A start job for unit systemd-tmpfiles-clean.service has finished successfully
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A start job for unit systemd-tmpfiles-clean.service has finished successfully.
--
-- The job identifier is 2409.
lines 3049-3097/3097 (END)
```

**Comment by r-saikat** [comment]
My bad. This is a common issue. I fixed this. If anyone else needs a solution, visit:

[https://medium.com/@niktrix/getting-rid-of-systemd-resolved-consuming-port-53-605f0234f32f](https://medium.com/@niktrix/getting-rid-of-systemd-resolved-consuming-port-53-605f0234f32f
)
[https://www.reddit.com/r/WireGuard/comments/ilz0qm/cant_get_wg_to_work_on_port_53/](https://www.reddit.com/r/WireGuard/comments/ilz0qm/cant_get_wg_to_work_on_port_53/)
